src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1553,7 +1555,7 @@ public void CheckLintErrorsAndWarnings ()
 			};
 			proj.UseLatestPlatformSdk = false;
 			proj.SetProperty ("AndroidLintEnabled", true.ToString ());
-			proj.SetProperty ("AndroidLintDisabledIssues", "StaticFieldLeak,ObsoleteSdkInt,AllowBackup");
+			proj.SetProperty ("AndroidLintDisabledIssues", "");
```

Comment:
Is this one supposed be blank now, or was that on accident?
```suggestion
			proj.SetProperty ("AndroidLintDisabledIssues", disabledIssues);
```
If you are doing that on purpose, looks good otherwise 👍